### PR TITLE
[Ja] Translate ja/controllers/components/csrf.rst

### PR DESCRIPTION
**Pull request by yurimoon**
http://book.cakephp.org/3.0/ja/controllers/components/csrf.html

### CODE REVIEW

**Comment by chinpei215** [comment]
@yurimoon You may need to add the `check-the-request` section into `ja/controllers/request-response.rst`, so that the build passes. May I do it for you?

(ビルドを通過させるために、`ja/controllers/request-response.rst` に `check-the-request` の節を追加する必要があるかもしれません。私が追加しましょうか？)

**Comment by okinaka** [comment]
@chinpei215 I added the check-the-request section in cakephp/3.0. We can merge this PR.

File: ja/controllers/components/csrf.rst [hunk 1]
```diff
@@ -1,16 +1,74 @@
...
+コンポーネントの配列に ``CsrfComponent`` を加えるだけで、CSRFから防御されます。::
+
+    public function initialize()
+    {
+        parent::initialize();
+        $this->loadComponent('Csrf');
+    }
+
+コンポーネントの設置箇所で、コンポーネントに設定値を渡すことができます。
+使用可能な設定オプションは以下の通りです。::
```
Comment (okinaka) [inline ja/controllers/components/csrf.rst]: Delete ::

File: ja/controllers/components/csrf.rst [hunk 2]
```diff
@@ -1,16 +1,74 @@
...
+CsrfComponentはユーザのブラウザにクッキーを設定することで動作します。
+フォームが :php:class:`Cake\\View\\Helper\\FormHelper` を用いて作成されている場合、hiddenフィールドにCSRFトークンが追加されます。
+``Controller.startup`` イベントの間、もしリクエストがPOST, PUT, DELETE, PATCHリクエストの場合は、コンポーネントはリクエストデータとクッキーの値を比較します。
+もし、どちらかがない場合、あるいは2つの値が一致しない場合は、コンポーネントは :php:class:`Cake\\Network\\Exception\\InvalidCsrfTokenException` を投げます。
 
 .. note::
-    The documentation is not currently supported in Japanese language for this
-    page.
+    副作用が起こる前に、HTTPメソッドが使われているかをいつも確かめてください。
+    正しいHTTPメソッドが使用されているかを確認するために、 :ref:`check the HTTP method <check-the-request>` または :php:meth:`Cake\\Network\\Request::allowMethod()` を使用してください。
```
Comment (okinaka) [inline ja/controllers/components/csrf.rst]: "check the HTTP method" は、翻訳しても良いと思います。

**Comment by okinaka** [comment]
@yurimoon Thanks!

The sphinx's errors are not related this PR.

**Pull request merged**